src/items/items.controller.spec.ts

```diff
@@ -13,9 +15,29 @@ describe('ItemsController', () => {
     }).compile();
 
     controller = module.get<ItemsController>(ItemsController);
+    itemsService = module.get<ItemsService>(ItemsService);
   });
 
-  it('should be defined', () => {
-    expect(controller).toBeDefined();
+  describe('createItem', () => {
+    it('should return success with valid body', async () => {
+      const requestBody = {
+        name: 'any_name',
+        points: 18,
+      };
+
+      const item: Item = {
+        id: 'any_id',
+        name: 'any_name',
+        points: 18,
+        createdAt: new Date(),
+      };
+
+      jest
+        .spyOn(itemsService, 'createItem')
+        .mockImplementation(() => Promise.resolve(item));
```

Comment:
Sugestão se quiser deixar mais simples, pode usar esse método
```suggestion
        .mockResolvedValueOnce(item)
```

Reply:
Boa, foi!